src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Withdraw.tsx

```diff
@@ -306,7 +316,7 @@ export const Withdraw: React.FC<WithdrawProps> = ({ accountId, fromAddress, onNe
   const safeOutboundFeeInAssetCryptoBaseUnit = useMemo(() => {
     if (!outboundFeeInAssetCryptoBaseUnit) return
     // Add 5% as as a safety factor since the dust threshold fee is not necessarily going to cut it
-    return bnOrZero(outboundFeeInAssetCryptoBaseUnit).times(1.05).toFixed()
+    return bnOrZero(outboundFeeInAssetCryptoBaseUnit).times(1.05).toFixed(0)
```

Comment:
The other reason why inputing exactly the minimum amount (or just around) would still fail validation - a base unit amount should have no decimal places, and this one had some.